debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -1077,9 +1089,11 @@ public void shouldRecoverFromRetriableException() throws Exception {
         // kill all opened connections to the database
         TestHelper.execute("SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE backend_type='walsender'");
         TestHelper.execute(INSERT_STMT);
+        TestHelper.waitFor(Duration.ofSeconds(10));
```

Comment:
Should we reduce the wait duration ?

Reply:
I have kept it like this to be on the safer side, we could reduce later once we gain more stability.